src/components/README.md

```diff
@@ -0,0 +1,26 @@
+- [PatientHeader](https://github.com/openmrs/openmrs-react-components/tree/OEUI_183/src/components#patientheader)
+
+
+# PatientHeader
+
+### Props
+
+
+| Name |Type | Default | Description  |
+| ------------- |:-------------:| --- | ---- |
+| patient|`object`|  |  Patient object having `person`, `patientId` and `attributes` properties |
+| note | `array` | `[]` | The first index is displayed, index will be an object having `value` and `auditInfo` properties|
+
+
+```
+import { PatientHeader } from '@openmrs-react-component';
+
+function header = (props) => {
+    return (
+        <PatientHeader
+          patient={props.patient}
+          note={props.note}
```

Comment:
This note thing is a little strange, but I realize just something you are copying over, I hadn't noticed it before... this is just more an fyi to myself to look into this and perhaps ticket future work.

Reply:
yeah, sure mark! Thanks.